js/src/gtag-events/constants.js

```diff
@@ -0,0 +1,3 @@
+export const namespace = 'google-listings-and-ads';
+export const actionPrefix = 'experimental__woocommerce_blocks';
+export const sendToGroup = 'GLA';
```

Comment:
💅 

To consider to use the UPPERCASE_SNAKE_CASE syntax for the constant names as we do in this `constants.js` [here](https://github.com/woocommerce/google-listings-and-ads/blob/develop/js/src/constants.js) for example